content/en/real_user_monitoring/platform/connect_rum_and_traces.md

```diff
@@ -524,6 +524,9 @@ Datadog uses the distributed tracing protocol and sets up the following HTTP hea
 : To make sure that the Agent keeps the trace.
 {{% /tab %}}
 {{% tab "W3C Trace Context" %}}
+
+The W3C Trace Context header is sent by default. This means you need to configure `traceparent` for CORS servers in addition to the default Datadog-specific headers.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Since by default both trace context and datadog headers are used, what about adding a note mentionning the two outside the tabs?

Reply:
Done, thanks!